package.json

```diff
@@ -81,6 +81,11 @@
     "nanoid": "3.1.31",
     "glob-parent": "6.0.2",
     "simple-get": "4.0.1",
-    "semver": "7.5.3"
+    "semver": "7.5.3",
+    "homedir-polyfill": "../_EXCLUDED_",
+    "inflight": "../_EXCLUDED_",
+    "rimraf": "5.0.7",
+    "@humanwhocodes/config-array": "@eslint/config-array",
+    "glob": "10.4.2"
```

Comment:
I did a search and didn't find any references to rimraf or config-array - how do you know they're needed? "glob" is difficult to search for - it hits on "global" - but I'm likewise wondering how we know this needs to be added.

Reply:
When we import the dependencies, npm identifies the deprecated ones and the ones no longer supported. The replacement was also suggested by npm.

Reply:
But do we actually need these? Unused dependencies only get more difficult to remove over time as it becomes more and more difficult to know if a dependency is needed or not. 

For example, I can't find any references to the string `rimraf` except in `package-lock.json`. That seems like a strong indicator that it's not needed. Maybe it still is, but I would opt for not including it and see if the Jenkins pipeline passes first. Same applies for `config-array`.

Reply:
config-array can be excluded but excluding rimraf throws error.